definitions/contested/json/CaseEvent/CaseEvent.json

```diff
@@ -999,5 +999,18 @@
     "SecurityClassification": "Public",
     "ShowSummary": "N",
     "ShowEventNotes": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "FR_updateFRCInformation",
+    "Name": "Update FRC Information",
+    "Description": "Update FRC Information",
+    "DisplayOrder": 140,
+    "PreConditionState(s)": "applicationIssued;gateKeepingAndAllocation;schedulingAndHearing;caseFileSubmitted;close;consentProcess;awaitingJudiciaryResponse;consentOrderApproved;consentOrderNotApproved;consentOrderMade;caseWorkerReview;judgeDraftOrder;solicitorDraftOrder;reviewOrder;draftOrderNotApproved;scheduleRaiseDirectionsOrder;orderDrawn;orderSent;consentedOrderSubmitted;awaitingJudiciaryResponseConsent;consentedOrderAssignJudge;consentedOrderApproved;responseReceived;consentedOrderNotApproved;generalApplication;generalApplicationAwaitingJudiciaryResponse;generalApplicationOutcome;prepareForHearing",
```

Comment:
seems like we have way too many states in this service ;D were any left? otherwise we could use *

Reply:
Yeah in this case we need to withhold the event from pre-submitted states, ccd should have some kind of functionality that's like "* _except_ a, b, c".. one can dream